integration-tests/config/ocr2_config.go

```diff
@@ -0,0 +1,195 @@
+package config
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+)
+
+type OCR2Config struct {
```

Comment:
Doesn't it make sense to move and organise these types in the `solana` Go module vs. `integration-tests`?

Reply:
I don't think it does because these are types the tests use to store information

the relayer does not use these types - it only uses types for parsing onchain data (sort of similar - but not really)

Reply:
some of these types are gauntlet types that are reimplemented in go